doc/api/next_api_changes/removals/20465-ES.rst

```diff
@@ -0,0 +1,91 @@
+Case-insensitive properties
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Upper or mixed-case property names are no longer normalized to lowercase in
+`.Artist.set` and `.Artist.update`. This allows one to pass names such as
+*patchA* or *UVC*.
+
+Case-insensitive capstyles and joinstyles
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Please pass capstyles ("miter", "round", "bevel") and joinstyles ("butt",
+"round", "projecting") as lowercase.
+
+AVConv animation writers removed
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The ``AVConvBase``, ``AVConvWriter`` and ``AVConvFileWriter`` classes, and the
+associated ``animation.avconv_path`` and ``animation.avconv_args`` rcParams
+have been removed.
+
+Debian 8 (2015, EOL 06/2020) and Ubuntu 14.04 (EOL 04/2019) were the
+last versions of Debian and Ubuntu to ship avconv.  It remains possible
+to force the use of avconv by using the FFmpeg-based writers with
+:rc:`animation.ffmpeg_path` set to "avconv".
+
+``MovieWriter`` attributes
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+* ``animation.html_args`` rcParam
```

Comment:
```suggestion
* The rcParam ``animation.html_args`` has been removed.
```
IMHO for clarity, we should take the effort and make full sentences. Also in the following bullet points.

Reply:
I'm a bit lazier here because a) this is in the removal section, so it's implied, and b) I re-edit these when I put together final release notes and they may or may not be re-grouped, and then will get an overall sentence then.